src/panels/config/js/yaml_textarea.tsx

```diff
@@ -14,12 +14,20 @@ const isEmpty = (obj) => {
 export default class YAMLTextArea extends Component<any, any> {
   constructor(props) {
     super(props);
-    this.state = {
-      isvalid: true,
-      value:
+
+    let value: string | undefined;
+    try {
+      value =
         props.value && !isEmpty(props.value)
           ? yaml.safeDump(props.value)
-          : undefined,
+          : undefined;
+    } catch (err) {
+      alert(`There is an error in the YAML: ${err}`);
```

Comment:
Not an error in the YAML, could not convert it to YAML .

Reply:
But we don't have any other way to save the config than YAML? So where could the error come from? YAML -> JSON -> YAML?

Reply:
Good question, I don't know :D

Reply:
It's fine.